src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -617,6 +629,31 @@
     text: az sql midb show -g mygroup --mi myinstance -n mymanageddb
 """
 
+helps['sql midb retention-policy'] = """
+type: group
+short-summary: Manage SQL managed instance database backup retention policy.
+"""
+
+helps['sql midb retention-policy update'] = """
+type: command
+short-summary: Update short term retention for automated backups on a single database.
+examples:
+  - name: Update backup short term retention for live managed database.
+    text: az sql midb retention-policy update -g mygroup --mi myinstance -n mymanageddb --retention-days retentionindays
+  - name: Update backup short term retention for dropped managed database.
+    text: az sql midb retention-policy update -g mygroup --mi myinstance -n mymanageddb --deleted-time "2018-05-20T05:34:22" --retention-days retentionindays
+"""
+
+helps['sql midb retention-policy show'] = """
```

Comment:
I think we need to be explicit in what retention-policy is being updated. Our naming convention is unfortunate, but it's best to make sure the apis and clients all align. From that, we should use short-term-retention-policy

Reply:
Sure.